client/src/components/CardList/CardList.stories.js

```diff
@@ -0,0 +1,37 @@
+import _ from "lodash";
+import React from "react";
+
+import { CardList } from "./CardList";
+
+export default {
+  title: "CardList",
+  component: CardList,
+};
+
+const Template = (args) => <CardList {...args} />;
+
+const elements = [
+  {
+    display: "Display 1",
+    href: "",
```

Comment:
Maybe name this one "Non-link title", leave `href` blank

Reply:
Done!